include/SSVOpenHexagon/Core/MenuGame.hpp

```diff
@@ -273,7 +273,7 @@ class MenuGame
 
     // Load menu
     const hg::HGAssets::LoadInfo& loadInfo;
-    std::vector<std::string> randomTip;
+    std::string randomTip[2];
```

Comment:
```suggestion
    std::array<std::string_view, 2> randomTip;
```

Please also include `<array>` and `<string_view>`. 

`std::array` is the same as a C-style array, but it has a nice container interface (e.g. `.size()`). `std::string_view` is a non-owning view over an existing range of characters which is stable in memory. Since our tips are `static`, we can use it instead.

Reply:
All taken care of.